CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
-## v1.1
+## v1.2.0
```

Comment:
I figured if we're versioning, we might as well go full [semver](https://semver.org/) here.

Reply:
Yes